### DISCUSSION TITLE
Changing the directory for userChromeJS scripts

### DISCUSSION THREAD

**Discussion by gh2402** [xiaoxiaoflood/firefox-scripts#315] [General Support] [answered]
Is there any way to change the directory that userChromeJS scripts are placed in? It seems that most scripts, either from xiaoxiaoflood or other authors, go in the user's <profile>/chrome directory. Can I make a subdirectory of /chrome and put scripts in that instead? I tried modifying the config.js file in the Firefox program dir, and the files in the utils dir, but couldn't get it to work. I don't know enough about Javascript to understand how those scripts work and how they find the user's chrome directory.

**Comment by xiaoxiaoflood** [comment, marked as answer]
https://github.com/xiaoxiaoflood/firefox-scripts/blob/0bbf504dedd793790e94dc88f8f90f3f18429132/chrome/utils/userChrome.jsm#L21

You can edit this line in this file, save then restart Firefox.

Example:

```js
scriptsDir: 'scripts',
```

Now Firefox will load scripts from `<profilefolder>/chrome/scripts`.

**Reply by gh2402** [reply]
Thank you so much, it works fine. I saw that line in the file earlier when I was trying different things, and couldn't figure out why it was set to an empty string.
